event.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Consider adding a schema definition for the JSON structure to ensure that the data adheres to expected formats and types.